portfolio/src/main/webapp/style.css

```diff
@@ -1,59 +1,120 @@
+#main-container{
```

Comment:
Noticed some repeating colors. Consider trying this - https://www.w3schools.com/css/css3_variables.asp :)